common/artifacts/000_wc_artifacts.txt

```diff
@@ -818,6 +1161,107 @@ chalice_of_trollbanes = {
 
 	picture = "GFX_inv_misc_trophy_argent"
 }
+orb_of_corruption = {
+	quality = 4
+	flags = { misc epic }
+
+	active = {
+		is_adult = yes
+	}
+
+	indestructible = yes
+
+	martial = 2
+	intrigue = 2
+
+	combat_rating = 10
+
+	monthly_character_prestige = 1
+
+	general_opinion = -10
+	fel_group_opinion = 20
+	death_god_opinion = 20
+
+	picture = "GFX_orb_of_corruption"
+	slot = libram
+}
+spellbook_of_drekthar = {
+	quality = 4
```

Comment:
Why it's epic? It stopped appearing in the lore after Warcraft 3. I would give it 3 quality.

Reply:
by values it's already closer to rare then to epic. and below planned average for epic

Reply:
But the point is still here. It's just random speelbook Drek'thar stole from dead Dalaran mage. I would even give it 2 tier to be honest.

Reply:
From how I understood the description, it's a book he made from spell pages he took from mages. So no standard book you can find everywhere but something a little more special, unique...

Reply:
> Drek'thar's Spellbook is an old and seemingly simple spellbook, filled with pages stolen from Kirin Tor mages that were slain in battle.

https://wow.gamepedia.com/Drek%27thar%27s_Spellbook

If this book is "seemingly simple" to Rexxar, I would say it's 1 tier.

Reply:
demoted to tier 3